src/sentry/static/sentry/app/redux/actions/shared.js

```diff
@@ -85,18 +84,64 @@ const acCreate = (resource, urlTemplate) => {
         dispatch(acCreateSuccess(resource)(data));
       },
       error: (err) => {
-        dispatch(acCreateFailure(resource)(err.statusCode, err.statusText));
+        const message = getErrorMessage(err);
+        dispatch(acCreateFailure(resource)(err.status, message));
       },
     });
     // return axios
     //   .post(url, data)
     //   .then(() => dispatch(acCreateSuccess(resource)(data)))
-    //   .catch((err) =>
-    //     dispatch(acCreateFailure(resource)(err.statusCode, err.statusText))
-    //   );
+    //   .catch((response) => {
```

Comment:
Not critical, but I think all these commented out axios calls should have a comment with the JIRA ticket number of the issue where we plan to fix these. Just for context.